languages/rust/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Tuples
+
+Tuples are a lightweight way to group a bounded set of heterogenous data together. A tuple doesn't have
+a particular name; naming a data structure turns it into a `struct`. A tuple's fields don't have names;
+they are accessed by means of destructuring or by position.
+
+## Syntax
+
+Tuples are a list of fields surrounded by parentheses:
+
+- `()` is the `unit type`, which is a [special case](https://doc.rust-lang.org/std/primitive.unit.html)
```

Comment:
Do students need to know this to solve the exercise? If not, it should be removed here and only included in the concept's `about.md` file.

Reply:
What's the difference between `languages/X/concepts/C/about.md` and `languages/X/reference/exercise-concepts/C.md`? Right now, this PR defines both for `tuples` and `destructuring`, which feels redundant.

Reply:
`languages/X/reference/exercise-concepts/C.md` is something that the v3 website doesn't care for. It is track-specific content just there for maintainers.